pyproject.toml

```diff
@@ -28,6 +29,7 @@ include = [
 python = "^3.8"
 napalm = {version = "^4.0.0", optional = true}
 jsonschema = {version = "^4.17.3", optional = true}
+looseversion = "^1.3"
```

Comment:
I was planning on vendoring this, you can remove this one as we don't have any core requirements. If you can figure out a workaround for [crypt](https://docs.python.org/3/library/crypt.html) that would be great :)

Reply:
Not sure I understand what you're recommending. Should I vendor the code instead of adding the dependency as a part of this PR?

Reply:
Looks like an alternative approach to this PR would be to add `setuptools` as an explicit dependency, as setuptools vendors `distutils` into itself.

Reply:
This should work: https://github.com/networktocode/netutils/pull/484

Reply:
Thanks, rebased this branch under yours.